nixos/modules/installer/sd-card/sd-image-aarch64-new-kernel-installer.nix

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
shouldn't this file be in the previous commit

Reply:
Fixed this truly, now. Thanks!